osu.Game/Screens/Edit/Editor.cs

```diff
@@ -1129,6 +1133,14 @@ private EditorMenuItem createExportMenu()
             return new EditorMenuItem(CommonStrings.Export) { Items = exportItems };
         }
 
+        private void editExternally()
+        {
+            Save();
```

Comment:
At the same time, this save usage probably isn't robust. The save function returns a boolean on whether the save was successful or not, there is no check here to ensure this.

It would be wise to switch to using
```suggestion
            if (!Save()) return;
```
or
```suggestion
            if (!Save()) throw new InvalidOperationException("Failed to save the beatmap.");
```
and have the function's caller catch this.